### DISCUSSION TITLE
Too many features

### DISCUSSION THREAD

**Discussion by SkewedZeppelin** [netblue30/firejail#5183] [Q&A] [answered]
The past few versions have been adding more and more arguable unnecessary features further increasing attack surface of firejail.

The best part of firejail is the collection of ready to use profiles for programs, nothing else offers this.
What ever happened to the minimal LTS version?

Can these be #ifdef'ed out?
- [x] ids is --disable-ids
- [ ] --appimage
- [x] --bandwidth, --netstats, --nettrace are --disable-network
- [ ] --build?
- [ ] --cpu?
- [x] --tunnel is --disable-firetunnel

any others?

**Comment by rusty-snake** [comment]
> Can these be #ifdef'ed out?

While for us #ifdef works good, the most users will get their firejail binary from their package-manager. Therefore we should make sure that there are options in firejail.config to disable those features too (--appimage has one).

> any others?

- [ ] --cgroup
- [ ] --shutdown
- [ ] --trace (subset of --build)
- [ ] --tracelog?

**Comment by netblue30** [comment, marked as answer]
moved to https://github.com/netblue30/firejail/issues/5190
